### ISSUE TITLE
Links to p5.js from Dodge & Soccer need to be https

### ISSUE THREAD

**Issue by jonleung** [hackclub/hackclub#356]
Change the script tags in the JS Bin and the Readme to be:

```html
  <script src="https://rawgit.com/jonleung/33aa8dcb08ce87d689a6/raw/9a3acc3f39ea114aadc1ab87c75f518c553f54ca/p5.js"></script>
  <script src="https://rawgit.com/jonleung/33aa8dcb08ce87d689a6/raw/9a3acc3f39ea114aadc1ab87c75f518c553f54ca/p5.play.js"></script>
```

These use

- `https://`
- Are the most up to date p5 libraries

**Comment by jonleung** [comment]
Closing this because this doesn't seem problematic enough.

**Comment by MaxWofford** [comment]
@jonleung Just a heads up, but https should be a standard for all our materials. (there aren't markcop tests because some things have to be http, but we should default to https)

**Comment by paked** [comment]
@MaxWofford what things have to be http? Would it make sense to work on a
whitelist basis as opposed to just letting everything through?

**Comment by MaxWofford** [comment]
@paked I don't know of anything in our workshops that have to be http, but some sites don't run https. We can probably set  something up when we run into the issue of linking something that doesn't support http in our workshops.

**Comment by jonleung** [comment]
I'm opting to leave this as is for now.

**Comment by jonleung** [comment]
I was thinking about it again, and I think it makes sense to leave this issue open.

I initially wanted to close it because I didn't want to prioritize this issue because I don't think fixing this it's very high leverage to our curriculum at the moment. However, it may make sense to have this fixed in the future, so I'm leaving re-opening this issue.

**Comment by MaxWofford** [comment]
I think this can be closed by: https://github.com/hackclub/hackclub/issues/556

**Comment by MaxWofford** [comment]
This seems like it can be done by a first time contributor. Adding the `first-timers-only` tag

**Comment by reicko** [comment]
https://github.com/hackclub/hackclub/pull/642

**Comment by zachlatta** [comment]
Closed by #642. Thank you @reicko!
